xbmc/platform/win32/storage/discs/Win32DiscDriveHandler.h

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "storage/discs/IDiscDriveHandler.h"
+
+#include <string>
+
+enum class CWin32DiscDriveState
+{
+  DRIVE_ERROR = -1,
```

Comment:
If you don't map between an int an an enum class above you don't need to initialize this enum with values

Reply:
Yeah leftover, they use to mean the libcdio return codes, will remove.